mods/ra/maps/drop-zone-w/weapons.yaml

```diff
@@ -2,42 +2,20 @@
 	ReloadDelay: 200
 	Range: 32c0
 	Burst: 4
-	Report: turret1.aud
 	Projectile: Bullet
 		Speed: 546
-		High: true
-		Angle: 62
+		LaunchAngle: 62
```

Comment:
This can also be removed. This value [inherits from Artillery](https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/weapons/ballistics.yaml#L96).